src/common/components/hosts/AITable.tsx

```diff
@@ -49,23 +50,25 @@ const TableMemo: React.FC<WithTestID & TableMemoProps> = React.memo(
       canCollapseAll: false,
     };
     return (
-      <Table
-        rows={rows}
-        cells={cells}
-        onCollapse={onCollapse}
-        variant={TableVariant.compact}
-        aria-label="Hosts table"
-        className={classnames(className, 'hosts-table')}
-        sortBy={sortBy}
-        onSort={onSort}
-        rowWrapper={rowWrapper}
-        data-testid={testId}
-        actionResolver={actionResolver ? tableActionResolver : undefined}
-        {...newProps}
-      >
-        <TableHeader />
-        <TableBody rowKey={rowKey} />
-      </Table>
+      <InnerScrollContainer>
```

Comment:
This component becomes "invisible" when switching to less wide screen sizes.
Is there something that could be done to avoid this? Maybe setting some CSS to have minimum height or something?

I know we don't commit to non-desktop viewports, but let's see if we can adjust it a bit better.

Reply:
Fixed

https://user-images.githubusercontent.com/87187179/200569560-e1ac48c0-51ab-4d5c-b4e7-5436360d1ecd.mp4